### PR TITLE
docs: 4.5.b edge known issues

### PR DESCRIPTION
**Pull request by lennessyy**
## Describe the Change

This PR adds the known issues for Edge in the 4.5.b release. 

## Changed Pages

💻 [Add Preview URL for Page]()

## Jira Tickets

🎫 [PE-5695](https://spectrocloud.atlassian.net/browse/PE-5695)

## Backports

Can this PR be backported?

- [ ] Yes. _Remember to add the relevant backport labels to your PR._
- [x] No. _Please leave a short comment below about why this PR cannot be backported._

[PE-5695]: https://spectrocloud.atlassian.net/browse/PE-5695?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/release-notes/known-issues.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ The following table lists all known issues that are currently active and affecti
...
...luster, resetting an in-use follower node does not update the node status on the leader node's linking screen.                                                                                                                                                                                                                                                                                                                                                                                                                                                           | [Scale down](../clusters/edge/local-ui/cluster-management/scale-cluster.md#scale-down-a-cluster) the cluster and free up the follower node before resetting the node.                                                                                                                                                                                                                                                                                                                                                                                                                                           | December 12, 2024  | Edge                         |
```
Comment (abhinavnagaraj) [inline docs/docs-content/release-notes/known-issues.md:20]: We can change "resetting an in-use follower" -> "the Edge Host reset action on a cluster node". It applies to all cluster nodes.

Comment (lennessyy) [inline docs/docs-content/release-notes/known-issues.md:20]:
```suggestion
| In a multi-node Edge cluster, the reset action on a cluster node does not update the node status on the leader node's linking screen.                                                                                                                                                                                                                                                                                                                                                                                                                                                           | [Scale down](../clusters/edge/local-ui/cluster-management/scale-cluster.md#scale-down-a-cluster) the cluster and free up the follower node before resetting the node.                                                                                                                                                                                                                                                                                                                                                                                                                                           | December 12, 2024  | Edge                         |
```

File: docs/docs-content/release-notes/known-issues.md [hunk 2]
```diff
@@ -16,6 +16,8 @@ The following table lists all known issues that are currently active and affecti
...
... in [agent mode](../deployment-modes/agent-mode/agent-mode.md) do not display host information in the console after the Palette Terminal User Interface completes host setup.                                                                                                                                                                                                                                                                                                                                                                                            | Local UI is still available and will display host information. Refer to [Access Local UI](../clusters/edge/local-ui/host-management/access-console.md) to learn how to access Local UI.                                                                                                                                                                                                                                                                                                                                                                                                                         | December 12, 2024  | Edge                         |
```
Comment (benradstone) [inline docs/docs-content/release-notes/known-issues.md:19]:
```suggestion
| Hosts provisioned in [agent mode](../deployment-modes/agent-mode/agent-mode.md) do not display host information in the console after using the Palette Terminal User Interface to complete host setup.                                                                                                                                                                                                                                                                                                                                                                                            | Local UI is still available and will display host information. Refer to [Access Local UI](../clusters/edge/local-ui/host-management/access-console.md) to learn how to access Local UI.                                                                                                                                                                                                                                                                                                                                                                                                                         | December 12, 2024  | Edge                         |
```

**Review by benradstone — APPROVED** [review]
Approved with one sentence change suggestion.

**Pull request merged**